tests/DbAuth.py

```diff
@@ -38,12 +38,11 @@ class DbAuthTestCase(unittest.TestCase):
     """A test case for DbAuth."""
 
     def setUp(self):
-        self.pol = Policy(os.path.join(ROOT, "testDbAuth.paf"))
```

Comment:
it was nonsensical to keep self.pol here; DbAuth has a static shared ptr where it gets kept, and calling reset releases/deletes it.